modules/cluster/addons.tf

```diff
@@ -95,12 +102,3 @@ module "nvidia_device_plugin" {
   replace  = true
 }
 
-module "aws_ebs_csi_driver" {
-  source = "./kubectl"
-  config = local.config
-  apply  = var.aws_ebs_csi_driver
```

Comment:
`var.aws_ebs_csi_driver` sets if this ebs driver should be used, or if we default to the in tree one, either we need to make the addon conditional on this, or remove the ability to choose which driver is used.

Reply:
fixed.  Use of CSI is optional, but if chosen to use it, should use the EKS addon.